src/components/overview/Overview.js

```diff
@@ -28,6 +29,7 @@ const [products, setProducts] = useState(false);
   //Api call to get the products array.
   return (
     <div>
+      {console.log('the products', products)}
```

Comment:
do you still need this console log?

Reply:
Yeah, I'm still not done with building out the components so I just left it in there for when I go back to working on it.